### PR TITLE
fix(internal/gapicgen): only open draft PR if there are genproto changes

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

**Comment by codyoss** [comment]
Example: #2633 opened as a draft when it should have just been a normal PR right away.

File: internal/gapicgen/cmd/genbot/github.go [hunk 1]
```diff
@@ -240,10 +240,12 @@ func (gc *GithubClient) CreateGocloudPR(ctx context.Context, gocloudDir string,
 	log.Println("creating google-cloud-go PR")
 
 	var body string
+	draft := github.Bool(true)
```
Comment (tbpg) [inline internal/gapicgen/cmd/genbot/github.go:243]: Consider making this a regular `bool` then converting it with `github.Bool` when calling `Create`.

File: internal/gapicgen/cmd/genbot/github.go [hunk 2]
```diff
@@ -240,8 +240,10 @@ func (gc *GithubClient) CreateGocloudPR(ctx context.Context, gocloudDir string,
 	log.Println("creating google-cloud-go PR")
 
 	var body string
+	var draft bool
```
Comment (tbpg) [inline internal/gapicgen/cmd/genbot/github.go:243]: Could just do `draft := false`, but it's functionally the same so whatever you think is easier to read.

Comment (codyoss) [reply internal/gapicgen/cmd/genbot/github.go:243]: Both seem equally readable to me, will keep the same for now.

**Pull request merged**